debugger.md

```diff
@@ -0,0 +1,81 @@
+﻿#Instructions for setting up the .NET Core debugger
+*Updated on  March 1st 2016 - setup works for Linux & Mac.*
+
+***Currently working within Microsoft Corp-Net only.***
```

Comment:
It seems okay for the hopefully two days we have this up. We don't have great ways to make something MS-specific but without access control.